developers/contributor-guide/weaviate-core/tests.md

```diff
@@ -87,29 +87,29 @@ go test ./...
 #### Adding new unit tests
 
 * Add unit tests in the same folder as the code they are testing
-* Aim to write "black box" unit tests which test the public ("exported") methods of the class under test without knowing too much about the internals
+* Aim to write "black box" unit tests that test the public ("exported") methods of the class under test without knowing too much about the internals.
 * Do not use any build tags.
 
 ### Integration tests
-As outline in the Philosophy, integration tests require backing services to be run. We have a convenience script available which starts all required services in `Docker Compose` and runs the tests against it:
+As outline in the Philosophy, integration tests require backing services to be run. We have a convenience script available which starts all required services in `docker compose` and runs the tests against it:
```

Comment:
Also  a product reference.

The product - command distinction would probably be clearer if we removed the monospacing from all of the references.